maintainers/scripts/get-nixos-module-maintainers/main.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
Wouldn't it be better to use `nix-shell` and declare the dependencies the scripts needs ?